Source/WebCore/rendering/RenderLayerFilters.cpp

```diff
@@ -169,10 +169,10 @@ GraphicsContext* RenderLayerFilters::beginFilterEffect(RenderElement& renderer,
     if (!m_filter)
         return nullptr;
 
-    auto& filter = *m_filter;
```

Comment:
It seems like the code seems to expect that filter cannot be null. Can/should we use `Ref` instead to keep this idea around?